pkg/extended_registry/aserto_test.go

```diff
@@ -0,0 +1,73 @@
+package extendedregistry
+
+import (
+	"net/http"
+	"os"
+	"testing"
+
+	"github.com/rs/zerolog"
+	"gopkg.in/h2non/gock.v1"
+	"gotest.tools/assert"
+)
+
+var opcrInfo = `
+{"info":{"Version":"v0.1.2","Date":"2022-02-11T11:55:20Z","Commit":"8005fd3"},"extended_api":"api.opcr.io","grpc_extended_api":"api.opcr.io:8443"}
+`
+
+func TestAsertoListOrgs(t *testing.T) {
+	testlog := zerolog.New(os.Stdout)
+	defer gock.Off() // Flush pending mocks after test execution
+	gock.New("https://opcr.io").
+		Get("/info").
+		Reply(200).BodyString(opcrInfo)
+	gock.New("https://opcr.io").
+		Get("/api/v1/registry/organizations").
+		Reply(200).BodyString("{}")
+	client := NewAsertoClient(&testlog, &Config{Address: "https://opcr.io", Username: username, Password: password}, http.DefaultClient)
+	orgs, err := client.ListOrgs()
+	assert.NilError(t, err)
+	t.Log(orgs)
```

Comment:
should this validate that at least one work is listed? add an assert for an org?